dcmgr/lib/dcmgr/rpc/windows_handler.rb

```diff
@@ -21,11 +21,18 @@ class WindowsHandler < EndpointBuilder
         [@hva_ctx]
       )
 
+      after_in_minutes = Dcmgr::Configurations.hva.windows.delete_password_after
+      if after_in_minutes == 0
+        expire = nil
+      else
+        expire = Time.now.utc + ( after_in_minutes * 60 )
+      end
+
```

Comment:
if administrator set minus number or string `delete_password_after param` in `hva.conf`, what will happen?